contra-tracer/contra-tracer.cabal

```diff
@@ -14,12 +14,12 @@ cabal-version:       >=1.10
 
 library
   hs-source-dirs:      src
-  exposed-modules:     Cardano.BM.Tracer
+  exposed-modules:     Control.Tracer
 
   other-modules:
-                       Cardano.BM.Tracer.Class
-                       Cardano.BM.Tracer.Output
-                       Cardano.BM.Tracer.Transformers
+                       Control.Tracer.Class
```

Comment:
We should rename this to `Type` since it's no longer a class. But actually, I think there'd be no harm at all if we just merged it all into the single top level module. It's not so big that it needs to be split out I think. But this is something we can change later, since these are now not public modules.